### ISSUE TITLE
Request returns 400 error

### ISSUE THREAD

**Issue by heitorsampaio** [ohld/igbot#1128]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- You will be asked some questions and requested to provide some information, please read them **carefully** and answer completely.
- Put an `x` into all the boxes [ ] relevant to your issue (like so [x]).
- Use the *Preview* tab to see how your issue will actually look like.

---

### Before submitting an issue, make sure you have:
- [X ] Updated to the latest version
- [X ] Read the [README](https://github.com/instagrambot/instabot/blob/master/README.md) and [docs](https://github.com/instagrambot/instabot/tree/master/docs)
- [X ] [Searched](https://github.com/instagrambot/instabot/search?type=Issues) the bugtracker for similar issues including **closed** ones

### Purpose of your issue?
- [ ] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [ X] Question
- [ ] Other

---

### The following sections requests more details for particular types of issues, you can remove any section (the contents between the triple ---) not applicable to your issue.

---

### For a *bug report*, you **must** include the Python version used, *code* that will reproduce the error, and the *error log/traceback*.

Paste the output of ``python -V`` here:

Code:

```python
# Example code that will produce the error reported
from instabot import Bot

bot = Bot()
bot.login(username="user", password="pass")
followers = bot.get_user_followers(bot.user_id)
```

Error/Debug Log:

```
Request returns 400 error, content: {"message": "feedback_required", "spam": true, "feedback_title": "Action Blocked", "feedback_message": "This action was blocked. Please try again later. We restrict certain content and actions to protect our community. Tell us if you think we made a mistake.", "feedback_url": "repute/report_problem/ig_direct_v2_thread_create/", "feedback_appeal_label": "Report problem", "feedback_ignore_label": "OK", "feedback_action": "report_problem", "status": "fail"}
```

---

### Describe your issue

Hey guys! is there any tip to bypass the bot filter of the instagram to not get 'banned' for spam?

**Comment by heitorsampaio** [comment]
I guess using proxy is one way?

**Comment by steffanjensen** [comment]
yes error 400 now cause blocks and should be evaded like the other block errors. I have seen this happens on watch_stories.

400 error is only causing blocks for a few functions. This should be implemented. @ohld 

These are the new error code, we have only implemented 404.
400 - Watch stories block
404 - Like Blocks / Follow blocks
429 - Scraping users blocks

**Comment by heitorsampaio** [comment]
Im getting this error in sending DM

**Comment by steffanjensen** [comment]
I see didnt know that, there are no errors on reply pending inbox.. it's unlimited. 

Good to know :)

These are the new error code, we have only implemented 404.
400 - Watch stories block/dm people
404 - Like Blocks / Follow blocks
429 - Scraping users blocks

**Comment by proxzi** [comment]
i have same problem
Request returns 400 error!
but when i want login

**Comment by reliefs** [comment]
that is a different issue and not related to this

**Comment by baptx** [comment]
I had this issue also when trying to send a message with IGdm: https://github.com/igdmapps/igdm/issues/1133
It looks like the account is banned until we give feedback to Instagram and tell them why it is blocked by mistake.
After I sent them a feedback using their official mobile app with Android x86 in VirtualBox, the message changed and it said the block expires in 1 week. But after sending a few messages without problem, it is blocked again.
Does someone know how we can intercept HTTPS requests of the Instagram Android app so we can contact the support without the Android app? It is not working with mitmproxy and frida, which is a known issue. The error message says we have to contact them at the URL repute/report_problem/ig_direct_v2_thread_create/ but we don't know how the data should be formatted.

**Comment by reliefs** [comment]
https://github.com/instagrambot/instabot/issues/1141#issuecomment-560955472

**Comment by bruvv** [comment]
You are soft banned. CLosing this in favor of this issue: https://github.com/instagrambot/instabot/issues/1194
